project-base/tests/App/Acceptance/acceptance.suite.yml

```diff
@@ -32,7 +32,7 @@ modules:
             populate: true
             cleanup: true
         Tests\App\Test\Codeception\Module\StrictWebDriver:
-            host: "%selenium_server_host%"
+            host: '%SELENIUM_SERVER_HOST%'
```

Comment:
this parameter is lowercased in `parameters_test.yaml.dist`. Is this change intentional?

Reply:
Yes it is intensional. This variable has value from ENV. It is weird but that is how the codeception works.

Anyway I have removed original parameter `selenium_server_host` because it is not used anymore.